patches/chrome-browser-ui-BUILD.gn.patch

```diff
@@ -28,3 +28,11 @@ index e5dd5217c3ed853370856b42fae7859033cd84e0..183b9b5d8a430a282228d81b359f9312
      }
    } else {  # 'OS!="win"
      # The Cast Linux build sets toolkit_views to false, but many files in this
+@@ -4298,6 +4302,7 @@ static_library("ui") {
+       "//ui/gfx/geometry",
+       "//ui/views:buildflags",
+     ]
++    deps += [ "//brave/vendor/bat-native-ledger:headers" ]
```

Comment:
this patch is not necessary, chrome/browser sources/deps are added through brave/browser/sources.gni